eva/optimizer/statement_to_opr_convertor.py

```diff
@@ -222,9 +223,29 @@ def visit_load_data(self, statement: LoadDataStatement):
         if table_metainfo is None:
             # Create a new metadata object
             table_metainfo = create_video_metadata(table_ref.table.table_name)
-            
+
+        self._populate_column_map(table_metainfo)
+
+        # if query had columns specified, we just copy them
+        if statement.column_list is not None:
+            column_list = statement.column_list
+        
+        # else we copy all the columns
+        else:
+            column_list = []
+            for key in self._column_map:
```

Comment:
Just realized, we have a bug here. The column_map might have columns from other tables as well. So, you can change the column_map to be a table_metadata_id -> column_name -> columns.